Gemfile.lock

```diff
@@ -270,6 +270,3 @@ DEPENDENCIES
   utf8-cleaner
   vcr
   webmock
-
-BUNDLED WITH
```

Comment:
If you update to the latest version of Bundler this will get generated automatically. I think we should keep this tracked or we're going to be constantly excluding these lines.